src/main/java/org/opensearch/security/util/KeyUtils.java

```diff
@@ -52,8 +52,8 @@ public JwtParserBuilder run() {
                 } else {
                     try {
                         PublicKey key = null;
-
-                        final String minimalKeyFormat = signingKey.replace("-----BEGIN PUBLIC KEY-----\n", "")
+                        final String minimalKeyFormat = signingKey.replaceAll("\\r|\\n", "")
```

Comment:
Do we want to use something like: https://docs.oracle.com/javase/9/docs/api/java/lang/System.html#lineSeparator-- instead? I think this is also fine though.

Reply:
Although this will also include test modifications so maybe more hassle than it is worth - the test change would maybe just be System.lineSeparator() + key, then the test passing on both linux and windows should be enough to test the validity of this fix.

Reply:
I thought about that, but everything that I have read indicated that this was a more general approach that is platform agnostic. This seemed to be the most agreed upon answer: https://stackoverflow.com/a/2163204